test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -52,6 +44,19 @@ std::string const basic_version_str = "VERSION"
                                       "test_parser version:"
                                       "SeqAn version: " + version_str;
 
+std::string license_text()
+{
+    std::ifstream license_file{std::string{{SEQAN3_TEST_LICENSE_DIR}} + "/LICENSE.md"};
+    std::stringstream buffer;
+    buffer << license_file.rdbuf();
+
+    std::string str = buffer.str();
```

Comment:
There is now problem with the implementation as is:

> 9) Behaves as an UnformattedOutputFunction. After constructing and checking the sentry object, checks if sb is a null pointer. If it is, executes `setstate(badbit)` and exits. Otherwise, extracts characters from the input sequence controlled by sb and inserts them into `*this` until one of the following conditions are met:
>   * end-of-file occurs on the input sequence;
>   * inserting in the output sequence fails (in which case the character to be inserted is not extracted);
>   * an exception occurs (in which case the exception is caught). 
> 
> If no characters were inserted, executes `setstate(failbit)`. If an exception was thrown while extracting, sets `failbit` and, if `failbit` is set in `exceptions()`, rethrows the exception.

https://en.cppreference.com/w/cpp/io/basic_ostream/operator_ltlt

See also
* https://stackoverflow.com/a/2602258/11996616
* https://insanecoding.blogspot.com/2011/11/how-to-read-in-file-in-c.html
* https://stackoverflow.com/a/4064681/11996616

Reply:
Alright, I did not know that. You should still do the checks whether opening the file was successful and whether the fialbit was set when inserting into the stringstream with <<

Reply:
Alright, I did not know that. You should still do the checks whether opening the file was successful and whether the fialbit was set when inserting into the stringstream with <<